### PR TITLE
[zh] Sync contribution guides (4)

### PR DESCRIPTION
**Pull request by tengqm**
This PR is about the new contents.

### CODE REVIEW

File: content/zh/docs/contribute/new-content/blogs-case-studies.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Have a look at the source for the
+[existing case studies](https://github.com/kubernetes/website/tree/master/content/en/case-studies).
+
+Refer to the [case study guidelines](https://github.com/cncf/foundation/blob/master/case-study-guidelines.md) and submit your request as outlined in the guidelines. 
+-->
+## 提交案例分析
+
+案例分析用来概述组织如何使用 Kubernetes 解决现实世界的问题。
+Kubernets 市场化团队和 {{< glossary_tooltip text="CNCF" term_id="cncf" >}} 成员
```
Comment (pigletfly) [inline content/zh/docs/contribute/new-content/blogs-case-studies.md:97]: Kubernets > Kubernetes

File: content/zh/docs/contribute/new-content/new-features.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+lists each feature that is planned for a release.
+Each line item includes the name of the feature, a link to the feature's main
+GitHub issue, its stability level (Alpha, Beta, or Stable), the SIG and
+individual responsible for implementing it, whether it
+needs docs, a draft release note for the feature, and whether it has been
+merged. Keep the following in mind:
+-->
+### 特性跟踪清单 {#feature-tracking-sheet}
+
+针对[给定 Kubernets 发行版本](https://github.com/kubernetes/sig-release/tree/master/releases)
```
Comment (pigletfly) [inline content/zh/docs/contribute/new-content/new-features.md:141]: Kubernets > Kubernetes 下同

Comment (tengqm) [reply content/zh/docs/contribute/new-content/new-features.md:141]: solved.

File: content/zh/docs/contribute/new-content/blogs-case-studies.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/contribute/new-content/blogs-case-studies.md:97]: fixed.

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 3]
```diff
@@ -0,0 +1,884 @@
...
+GitHub](#changes-using-github) to learn how to edit a page.
+
+If your changes are large, read [Work from a local fork](#fork-the-repo) to
+learn how to make changes locally on your computer.
+-->
+如果你所提交的变更足够小，或者你对 git 工具不熟悉，可以阅读
+[使用 GitHub 提交变更](#changes-using-github)以了解如何编辑页面。
+
+如果所提交的变更较大，请阅读[基于本地克隆副本开展工作](#fork-the-repo)以学习
+如果在你本地计算机上构造变更。
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:50]:
```suggestion
如何在你本地计算机上构造变更。
```

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 4]
```diff
@@ -0,0 +1,884 @@
...
+   <!--
+   This makes sure your local repository is up to date before you start making changes.
+   -->
+   这样可以确保你本地的仓库在开始工作前是最新的。
+
+   <!--
+    This workflow is different than the [Kubernetes Community GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md). You do not need to merge your local copy of `master` with `upstream/master` before pushing updates to your fork.
+   -->
+    {{< note >}}
+    此工作流程与 [Kubernetes 社区 GitHub 工作流](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)有所不同。在推送你的变更以更新的克隆版本之前，你不需要将你本地的 `master` 与 `upstream/master` 合并。
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:244]:
```suggestion
    此工作流程与 [Kubernetes 社区 GitHub 工作流](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)有所不同。在推送变更到你的远程派生副本库之前，你不需要将你本地的 `master` 与 `upstream/master` 合并。
```

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 5]
```diff
@@ -0,0 +1,884 @@
...
+      hugo server
+      ```
+<!--
+3.  In your browser’s address bar, enter `https://localhost:1313`.
+4.  To stop the local Hugo instance, go back to the terminal and type `Ctrl+C`,
+    or close the terminal window.
+-->
+3. 在浏览器的地址栏输入： `https://localhost:1313`。
+4. 要停止本地 Hugo 实例，返回到终端窗口并输入 `Ctrl+C` 或者关闭终端窗口。
+    or close the terminal window.
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:495]:
```suggestion
```

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 6]
```diff
@@ -0,0 +1,884 @@
...
+2. 点击 **New Pull Request**；
+3. 选择 **compare across forks**；
+4. 从 **head repository** 下拉菜单中，选取你的派生仓库；
+5. 从 **compare** 下拉菜单中，选择你的分支；
+6. 点击 **Create Pull Request**；
+7. 为你的拉取请求添加一个描述：
+    - **Title** （不超过 50 个字符）：总结变更的目的；
+    - **Description**：给出变更的详细信息；
+      - 如果存在一个相关联的 GitHub Issue，可以在描述中包含 `Fixes #12345` 或
+        `Closes #12345`。GitHub 的自动化设施能够当前 PR 被合并时自动关闭所提及
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:530]:
```suggestion
        `Closes #12345`。GitHub 的自动化设施能够在当前 PR 被合并时自动关闭所提及
```

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 7]
```diff
@@ -0,0 +1,884 @@
...
+<!--
+If your PR has multiple commits, you must squash them into a single commit before merging your PR. You can check the number of commits on your PR's **Commits** tab or by running the `git log` command locally.
+-->
+如果你的 PR 包含多个提交（commits），你必须将其压缩成一个提交才能被合并。
+你可以在 PR 的 **Commits** Tab 页面查看提交个数，也可以在本地通过
+`git log` 命令查看提交个数。
+
+<!-- This topic assumes `vim` as the command line text editor.-->
+{{< note >}}
+本主题嘉定使用 `vim` 作为命令行文本编辑器。
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:752]:
```suggestion
本主题假定使用 `vim` 作为命令行文本编辑器。
```

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 8]
```diff
@@ -0,0 +1,884 @@
...
+<!--
+Each repository has its own processes and procedures. Before you file an
+issue or submit a PR, read that repository's `README.md`, `CONTRIBUTING.md`, and
+`code-of-conduct.md`, if they exist.
+
+Most repositories use issue and PR templates. Have a look through some open
+issues and PRs to get a feel for that team's processes. Make sure to fill out
+the templates with as much detail as possible when you file issues or PRs.
+-->
+每个仓库尤其自己的流程和过程。在登记 Issue 或者发起 PR 之前，记得阅读仓库的
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/open-a-pr.md:871]:
```suggestion
每个仓库有其自己的流程和过程。在登记 Issue 或者发起 PR 之前，记得阅读仓库的
```

File: content/zh/docs/contribute/new-content/overview.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+- The source is in [GitHub](https://github.com/kubernetes/website). You can find Kubernetes documentation at `/content/en/docs/`. Some of the reference documentation is automatically generated from scripts in the `update-imported-docs/` directory.
+- [Page content types](/docs/contribute/style/page-content-types/) describe the presentation of documentation content in Hugo.
+- In addition to the standard Hugo shortcodes, we use a number of [custom Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/) in our documentation to control the presentation of content.
+- Documentation source is available in multiple languages in `/content/`. Each language has its own folder with a two-letter code determined by the [ISO 639-1 standard](https://www.loc.gov/standards/iso639-2/php/code_list.php). For example, English documentation source is stored in `/content/en/docs/`.
+- For more information about contributing to documentation in multiple languages or starting a new translation, see [localization](/docs/contribute/localization).
+-->
+## 基本知识
+
+- 使用 Markdown 来编写 Kubernetes 文档并使所有 [Hugo](https://gohugo.io/)
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/overview.md:36]:
```suggestion
- 使用 Markdown 来编写 Kubernetes 文档并使用 [Hugo](https://gohugo.io/)
```

File: content/zh/docs/contribute/new-content/overview.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  来构建网站
+- 源代码位于 [GitHub](https://github.com/kubernetes/website) 仓库中。
+  你可以在 `/content/en/docs/` 目录下找到 Kubernetes 文档。
+  某些参考文档是使用位于 `update-imported-docs/` 目录下的脚本自动生成的。
+- [页面内容类型](/docs/contribute/style/page-content-types/)使用 Hugo 描述文档内容的表现。
+- 除了基本的 Hugo 短代码（shortcodes）外，我们还在文档中使用一些
+  [定制的 Hugo 短代码](/docs/contribute/style/hugo-shortcodes/)以控制内容的表现。
+- 文档的源代码有多种语言形式，位于`/content/` 目录下。
+  每种语言都有自己的由两个字母代表的目录下，这两个字母是基于
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/overview.md:45]:
```suggestion
  每种语言都有自己的由两个字母代表的目录，这两个字母是基于
```

File: content/zh/docs/contribute/new-content/overview.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+  [ISO 639-1 标准](https://www.loc.gov/standards/iso639-2/php/code_list.php)来确定的。
+  例如，英语文档源码位于`/content/en/docs/` 目录下。
+- 关于在多种语言中为文档做贡献的详细信息，以及如何启动一种新的语言翻译，
+  克参考[本地化](/docs/contribute/localization)文档。
```
Comment (SataQiu) [inline content/zh/docs/contribute/new-content/overview.md:49]:
```suggestion
  参考[本地化](/docs/contribute/localization)文档。
```

**Comment by tengqm** [comment]
@SataQiu Thanks for the review. All feedbacks addressed. Sorry for so many typos. The Chinese IME on Mac really sucks.

**Pull request merged**